### PR TITLE
Track eks-distro-base image in separate file

### PR DESCRIPTION
**Pull request by abhay-krishna**
Tracking the tag for EKS Distro base image in a separate file, so that automatic PR logic will result in lighter PR's since just one file will change and not 12+ Makefiles.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -4,7 +4,7 @@ DEVELOPMENT?=false
 AWS_ACCOUNT_ID?=$(shell aws sts get-caller-identity --query Account --output text)
 AWS_REGION?=us-west-2
 IMAGE_REPO?=$(AWS_ACCOUNT_ID).dkr.ecr.$(AWS_REGION).amazonaws.com
-BASE_IMAGE?=$(AWS_ACCOUNT_ID).dkr.ecr.$(AWS_REGION).amazonaws.com/eks-distro/base:2021-01-05-1609822390
+BASE_IMAGE?=$(AWS_ACCOUNT_ID).dkr.ecr.$(AWS_REGION).amazonaws.com/eks-distro/base:$(shell cat EKS_DISTRO_BASE_TAG_FILE)
```
Comment (vivek-koppuru) [inline Makefile:7]: Let's separate out the variable and have that overridable instead, like BASE_IMAGE_TAG or BASE_TAG like below

**Review by vivek-koppuru — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (abhay-krishna) [reply Makefile:7]: Sure

**Review by vivek-koppuru — APPROVED** [review]
/lgtm

**Pull request merged**